compat_policy_test.go

```diff
@@ -0,0 +1,44 @@
+package nftables
+
+import (
+	"testing"
+
+	"github.com/google/nftables/expr"
+	"github.com/google/nftables/xt"
+	"golang.org/x/sys/unix"
+)
+
+func TestGetCompatPolicy(t *testing.T) {
```

Comment:
Can you add another test that installs an nftables rule which requires a compat policy?

Reply:
done

Reply:
add now